src/internal/deploy/cmds/ide.go

```diff
@@ -0,0 +1,215 @@
+package cmds
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+
+	docker "github.com/fsouza/go-dockerclient"
+	"github.com/pachyderm/pachyderm/v2/src/auth"
+	"github.com/pachyderm/pachyderm/v2/src/client"
+	"github.com/pachyderm/pachyderm/v2/src/enterprise"
+	"github.com/pachyderm/pachyderm/v2/src/internal/cmdutil"
+	"github.com/pachyderm/pachyderm/v2/src/internal/config"
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/grpcutil"
+	"github.com/pachyderm/pachyderm/v2/src/internal/helm"
+	"github.com/spf13/cobra"
+)
+
+const (
+	defaultIDEHubImage  = "pachyderm/ide-hub"
+	defaultIDEUserImage = "pachyderm/ide-user"
+
+	defaultIDEVersion      = "2.0.0-a2"
+	defaultIDEChartVersion = "0.9.1" // see https://jupyterhub.github.io/helm-chart/
+	ideNotes               = `
+	Thanks for installing the Pachyderm IDE!
```

Comment:
These leading spaces end up in the output, unless you go out of your way to trim them.  I usually just dispense with the indenting (which is what the old code did), but:

```go
const (
   ideNotes = strings.Join([]string{
       "This is line one!"
       "This is line two!"
    }, "\n") 
)
```

is an alternative.